docs/Changelog.md

```diff
@@ -14102,6 +14102,63 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="MeanSquaredError-12"></a>**MeanSquaredError-12**</a>
+
+  Loss function that measures the
+  mean squared error (squared L2 norm) between each element in the 'scores'
+  and 'labels'.
+  
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), l_n = (score_n - label_n)^2
+  , where N is the batch size.
+  
+  If 'weights' is provided, it should be broadcastable to shape of 'scores'.
+      L = Mul(weights, L)
+  , where Mul is element-wise binary multiplication with Numpy-style broadcasting support.
+  
+  Finally, L is reduced:
+  L = ReduceSum(L), if reduction = 'sum';
+      ReduceMean(L), if reduction = 'mean';
```

Comment:
Is weights allowed for 'mean'? If it is allowed, does it mean compute a weighted-mean (which would required dividing by the sum-of-weights, and not the batch-size)?

Reply:
weights are used only in tensorflow and are allowed for mean, and it is divided by batch-size